angular-library/package.json

```diff
@@ -1,6 +1,8 @@
 {
   "name": "@justifi/angular-components",
   "version": "4.4.0",
```

Comment:
Should the version get bumped to a patch version?

Reply:
Yes, indeed.